content/blog/2019/07/2019-07-04-cloudfeature-gsoc-cp1.adoc

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "GSoC Project Progress : Cloud features for external workspace manager"
+tags:
+- jenkins
+- gsoc
+author: yufei_zhang
```

Comment:
This is a case for supporting multiple authors in the blogpost

Reply:
@oleg-nenashev I looked at the haml, but I do not know how to add that feature. It would take me a some of time.

Reply:
Let’s assign to @oleg-nenashev who knows how to add multiple authors in haml

Reply:
I assigned this PR to @oleg-nenashev to add haml syntax usage